examples/hypervisor_manager.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"fmt"
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"github.com/HewlettPackard/oneview-golang/utils"
+	"os"
+)
+
+func main() {
+	var (
+		ClientOV                        *ov.OVClient
+		hypervisor_manager              = "172.18.13.11"
+		hypervisor_manager_display_name = "HM2"
+		//hypervisor_manager_2 = "eth88"
+	)
+	ovc := ClientOV.NewOVClient(
+		os.Getenv("ONEVIEW_OV_USER"),
+		os.Getenv("ONEVIEW_OV_PASSWORD"),
+		os.Getenv("ONEVIEW_OV_DOMAIN"),
+		os.Getenv("ONEVIEW_OV_ENDPOINT"),
+		false,
+		800,
+		"")
+	ovVer, _ := ovc.GetAPIVersion()
+	fmt.Println(ovVer)
+	initialScopeUris := &[]utils.Nstring{utils.NewNstring("/rest/scopes/03beb5a0-bf48-4c43-94a5-74b7b5de1255")}
+	hypervisorManager := ov.HypervisorManager{DisplayName: "HM1", Name: "172.18.13.11", Username: "dcs", Password: "dcs", Port: 443, InitialScopeUris: *initialScopeUris, Type: "HypervisorManagerV2"}
```

Comment:
Intend the initialization line by line so it's readable for the customers.

Reply:
Donr